content/en/blog/2024/getting-started-with-otelsql/index.md

```diff
@@ -0,0 +1,428 @@
+---
+title:
+  Getting Started with otelsql, the OpenTelemetry instrumentation for Go SQL
+linkTitle: Getting Started with otelsql
+date: 2024-02-24
```

Comment:
```suggestion
date: 2024-03-04
```

Please keep this suggestion open such that we set the right publication day before merging